docs/tutorials/tracking/SiamRPN_demo.py

```diff
@@ -0,0 +1,81 @@
+"""01. Predict with pre-trained SiamRPN models
+==========================================
+This article shows how to play with pre-trained SiamRPN models with only a few
+lines of code.
+First let's import some necessary libraries:
+"""
+
+from gluoncv import model_zoo, data, utils
+from gluoncv.utils.siamrpn_tracker import SiamRPNTracker as build_tracker
+import cv2
+import numpy as np
+import mxnet as mx
+import os
+from mxnet import nd
+import argparse
+######################################################################
+# Load a pretrained model
+# -------------------------
+#
+# Let's get an siamrpn model trained 
+# dataset with Alexnet as the base model. By specifying
+# ``pretrained=True``, it will automatically download the model from the model
+# zoo if necessary. For more pretrained models, please refer to
+# :doc:`../../model_zoo/index`.
+net = model_zoo.get_model('siamrpn_alexnet_v2_otb15', ctx = mx.cpu(0), pretrained=True)
+
+######################################################################
+# Build a pretrained model
+# -------------------------
+tracker = build_tracker(net)
+######################################################################
+# Pre-process an video 
+# --------------------
+#
+# Next we download an video，it as input to the network
+# gt_bbox is first frame object coordinates，and it is bbox(center_x,center_y,center_w,center_h)
+# this model has two module，one is tracker.init. it has Picture and coordinates of the previous frame as input.
+# the other is tracker.track. it has Picture of the predict frame needed as input.
+# And returns is Dictionaries. keys are gt_bbox and score.Represents the coordinates and scores of the predicted frame.
+
+im_video = utils.download('https://raw.githubusercontent.com/FrankYoungchen/siam_data/master/Dog.mp4')
+gt_bbox = [74, 86, 56, 48]
+
+def parse_args():
```

Comment:
Since this is only a demo, it is better to make it as short as possible, so you can just define `result_path = './result'`. There is no need to use argparse.